package/libsolv.spec.in

```diff
@@ -15,122 +15,97 @@
 # Please submit bugfixes or comments via http://bugs.opensuse.org/
 #
 
-Name:           libsolv
-Version:        @VERSION@
-Release:        0
-Url:            https://github.com/openSUSE/libsolv
-Source:         libsolv-%{version}.tar.bz2
-BuildRoot:      %{_tmppath}/%{name}-%{version}-build
-
-%bcond_without enable_static
-%bcond_without disable_shared
-%bcond_without perl_binding
-%bcond_without python_binding
-%bcond_without python3_binding
-%bcond_without ruby_binding
-%bcond_with zypp
 
+%define libname libsolv@LIBSOLV_SOVERSION@
```

Comment:
Use `%global` instead of `%define` to avoid weird scoping issues... 

c.f.: https://fedoraproject.org/wiki/Packaging:Guidelines#.25global_preferred_over_.25define

Reply:
Does not matter, both are inherited fine, only if you do scope defines it explodes.